charts/kube-prometheus-stack/values.yaml

```diff
@@ -2377,13 +2377,13 @@ prometheusOperator:
     enableProbe: false
 
     # resource config for prometheusConfigReloader
-    resources:
-      requests:
-        cpu: 200m
-        memory: 50Mi
-      limits:
-        cpu: 200m
-        memory: 50Mi
+    resources: {}
```

Comment:
The field cannot be set like this without further changes in the operator's deployment template. 

`resources` in this context gets used in a different way than `resources` in a container context: each single one gets inserted in the corresponding operator's flag, e.g.
```
--config-reloader-cpu-limit={{ .Values.prometheusOperator.prometheusConfigReloader.resources.limits.cpu }}
```

With the field set to null, a nil pointer error would occur.

Reply:
Thanks for pointing that out @zeritti 
Should be fixed now...